source/whats-new.txt

```diff
@@ -12,15 +12,33 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 2.0 <mongodb-analyzer-whats-new-2.0>`
 * :ref:`Version 1.5 <mongodb-analyzer-whats-new-1.5>`
 * :ref:`Version 1.4 <mongodb-analyzer-whats-new-1.4>`
 
+.. _mongodb-analyzer-whats-new-2.0:
+
+What's New in 2.0
+-----------------
+
+The 2.0 {+product+} release includes the following features,
+improvements, and fixes:
+
+- Supports only {+driver-short+} v3.0 and higher.
+- Drops support for the ``MALinq2002`` rule, since {+driver-short+} v3.0
+  removes the ``LINQ2`` provider.
```

Comment:
i: [since](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#s:~:text=press%20Enter.-,since,-Use%20since%20to)

```suggestion
- Drops support for the ``MALinq2002`` rule because {+driver-short+} v3.0
  removes the ``LINQ2`` provider.
```